feat: add new assets and enhance UI components

yikZero opened the pull request:
- Added chrome.png and hardwallet_together_logo.png assets for improved branding.
- Enhanced UniversalSearchInput component with loading state indicators for better user experience.
- Updated ExternalWalletList and OneKeyWalletConnectionOptions components to improve interactivity with cursor and hover styles.
- Refactored WebDappEmptyView to include responsive design adjustments and improved button interactions.
- Integrated new translation keys for enhanced localization support across multiple languages.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* New Features
  * Search input now shows a loading indicator.
  * Wallet connection options feature richer avatars and a dynamic subtitle for the extension status.
  * Home page content is now scrollable while remaining centered.

* Style
  * Added hover and pointer interactions to wallet lists and buttons.
  * Refreshed empty view: consolidated search, new “Or” divider, full-width actions, and enhanced EthDev badge interactivity.

* Chores
  * Added new translations for “Track” and the swap recipient popover.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

yikZero committed:
feat: add new assets and enhance UI components

- Added chrome.png and hardwallet_together_logo.png assets for improved branding.
- Enhanced UniversalSearchInput component with loading state indicators for better user experience.
- Updated ExternalWalletList and OneKeyWalletConnectionOptions components to improve interactivity with cursor and hover styles.
- Refactored WebDappEmptyView to include responsive design adjustments and improved button interactions.
- Integrated new translation keys for enhanced localization support across multiple languages.

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/7d1fd2b1-8555-42a1-9d4a-c2ea630bae36?refs=5ab55ac9835d78d3b8a56871f49a71768ddee5d9&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/7d1fd2b1-8555-42a1-9d4a-c2ea630bae36/license?refs=5ab55ac9835d78d3b8a56871f49a71768ddee5d9&source=prChecksComment)

yikZero committed:
Merge branch 'x' into web-dapp

originalix commented on packages/kit/src/components/WebDapp/WebDappEmptyView.tsx:
```diff
@@ -227,38 +207,34 @@ function WebDappEmptyView() {
+                    address,
+                    networkId: getNetworkIdsMap().eth, // Default to ETH for direct address input
+                  },
+                );
```
去除这个 props 的传参，只能通过点击 popover 的结果访问

yikZero committed:
refactor: remove address selection handler from WebDappEmptyView

- Removed the onAddressSelect handler from the WebDappEmptyView component to streamline functionality and improve code clarity.

huhuanming committed:
Merge branch 'x' into web-dapp

yikZero committed:
Merge branch 'x' into web-dapp

originalix committed:
Merge branch 'x' into web-dapp
